.github/workflows/ocsp-standalone-test.yml

```diff
@@ -181,96 +242,256 @@ jobs:
-          # standalone OCSP should return security domain info
+          # create CRL issuing point with the PKCS #7
+          docker exec ocsp pki-server ocsp-crl-issuingpoint-add \
+              --cert-chain $SHARED/ca_signing.p7
 
-      - name: Check OCSP publishing in CA
+      - name: Configure CRL publishing in CA
         run: |
-          # OCSP publishing should not be configured
-          docker exec ca pki-server ca-config-find | grep ^ca.publish. > output
+          # configure OCSP publisher
+          docker exec ca pki-server ca-config-set ca.publish.publisher.instance.OCSPPublisher.enableClientAuth true
+          docker exec ca pki-server ca-config-set ca.publish.publisher.instance.OCSPPublisher.host ocsp.example.com
+          docker exec ca pki-server ca-config-set ca.publish.publisher.instance.OCSPPublisher.nickName subsystem
+          docker exec ca pki-server ca-config-set ca.publish.publisher.instance.OCSPPublisher.path /ocsp/agent/ocsp/addCRL
+          docker exec ca pki-server ca-config-set ca.publish.publisher.instance.OCSPPublisher.pluginName OCSPPublisher
+          docker exec ca pki-server ca-config-set ca.publish.publisher.instance.OCSPPublisher.port 8443
 
-          echo -n > expected
-          sed -n '/^ca.publish.enable=/p' output | tee actual
-          diff expected actual
+          # configure CRL publishing rule
+          docker exec ca pki-server ca-config-set ca.publish.rule.instance.OCSPRule.enable true
+          docker exec ca pki-server ca-config-set ca.publish.rule.instance.OCSPRule.mapper NoMap
+          docker exec ca pki-server ca-config-set ca.publish.rule.instance.OCSPRule.pluginName Rule
+          docker exec ca pki-server ca-config-set ca.publish.rule.instance.OCSPRule.publisher OCSPPublisher
+          docker exec ca pki-server ca-config-set ca.publish.rule.instance.OCSPRule.type crl
 
-          echo -n > expected
-          sed -n '/^ca.publish.publisher.instance.OCSPPublisher-/p' output | tee actual
-          diff expected actual
+          # enable CRL publishing
+          docker exec ca pki-server ca-config-set ca.publish.enable true
 
-          echo -n > expected
-          sed -n '/^ca.publish.rule.instance.ocsprule-/p' output | tee actual
-          diff expected actual
+          # set buffer size to 0 so that revocation will take effect immediately
+          docker exec ca pki-server ca-config-set auths.revocationChecking.bufferSize 0
 
-      - name: Gather artifacts
-        if: always()
+          # update CRL immediately after each cert revocation
+          docker exec ca pki-server ca-config-set ca.crl.MasterCRL.alwaysUpdate true
+
+          docker exec ca pki-server ca-redeploy --wait
+
+      - name: Check cert revocation with CRL publishing
         run: |
-          tests/bin/ds-artifacts-save.sh ds
-          tests/bin/pki-artifacts-save.sh ca
-          tests/bin/pki-artifacts-save.sh ocsp
-        continue-on-error: true
+          # create cert request
```

Comment:
In line 355 there was a certificate on hold before CRL get published to OCSP and the OCSP query returned `unknown`. After CRL get published I am expecting to result `revoked` but there is no test so it should be added before verify with new certificate.

Reply:
OK, let me add it.